src/types.ts

```diff
@@ -120,3 +120,18 @@ export type SlackAuthResponse = {
   };
   error?: string;
 };
+
+export enum ContentLanguage {
```

Comment:
More of a generic note do we want to stick with singlular descriptions for some?
(not sure what yggdrasil sets)

but would assume even german would be `de-DE` for instance?

Reply:
no, this is how yggdrasil assigns it already